tests/test_verify_base_url.py

```diff
@@ -4,6 +4,8 @@
 
 import pytest
 
+from requests.packages.urllib3.util.retry import Retry
```

Comment:
Yes, though I wouldn't do too much matching of the error message. Just the exception class is probably sufficient.